light-rest-4j/src/main/java/com/networknt/codegen/rest/OpenApiGenerator.java

```diff
@@ -82,23 +82,23 @@ public String getFramework() {
     @Override
     public void generate(String targetPath, Object model, Any config) throws IOException {
         // whoever is calling this needs to make sure that model is converted to Map<String, Object>
-        String rootPackage = config.toString("rootPackage");
-        String modelPackage = config.toString("modelPackage");
-        String handlerPackage = config.toString("handlerPackage");
+        String rootPackage = config.toString("rootPackage").trim();
+        String modelPackage = config.toString("modelPackage").trim();
+        String handlerPackage = config.toString("handlerPackage").trim();
         
         boolean overwriteHandler = config.toBoolean("overwriteHandler");
         boolean overwriteHandlerTest = config.toBoolean("overwriteHandlerTest");
         boolean overwriteModel = config.toBoolean("overwriteModel");
         generateModelOnly = config.toBoolean("generateModelOnly");
         
         boolean enableHttp = config.toBoolean("enableHttp");
-        String httpPort = config.toString("httpPort");
+        String httpPort = config.toString("httpPort").trim();
         boolean enableHttps = config.toBoolean("enableHttps");
-        String httpsPort = config.toString("httpsPort");
+        String httpsPort = config.toString("httpsPort").trim();
```

Comment:
While a validation check can be added, where we wish to enforce the fact that both can't be empty, it is not for this Git Issue to address.

If they are not set, the value is "".
---------------------
This is a separate issue and, if @stevehu and yourself @NicholasAzar agree, we can create a separate issue and enforce this.
We need to remember to document it. 

Now, as this is codegen, maybe we can just default the https is enabled and set 8443, in case nobody sets it in config.json.